python/src/opendp/_lib.py

```diff
@@ -124,6 +124,42 @@ def next_raw(_voidp):
 except ImportError:  # pragma: no cover
     pass
 
+pl = import_optional_dependency("polars", raise_error=False)
+if pl is not None:
```

Comment:
May have come up elsewhere, but it might be nice to have a little context manager for places like this. Will link an issue to comment.